plugins/server.go

```diff
@@ -54,7 +54,7 @@ type Server struct {
 
 // MustRegister registers the Checkable with services.Checker, or exits upon failure.
 func (s *Server) MustRegister(name string, c services.Checkable) {
```

Comment:
If `name` has moved inside the `Checkable` type, can we simplify this API to

```
func (s *Server) MustRegister(c services.Checkable) {
	err := s.Register(c)
	if err != nil {
		s.Logger.Fatalf("Failed to register %s with health checker: %v", c.Name(), err)
	}
```

Reply:
Yeah I missed this one. Updated.